src/js/app.js

```diff
@@ -0,0 +1,21 @@
+$(document).ready(function() {
+  $("#search").on("click", function(e) {
+    e.preventDefault();
+    var storedQuery = $("#query").val();
+    var url = "http://www.reddit.com/search.json?q=" + storedQuery + "+nsfw:no";
+      console.log(url);
+    $.get(url, function(data1) {
+      // $("query").html()
+      for (var i = 0; i < data1.data.children.length; i++) {
+        var thumbnail = data1.data.children[i].data.thumbnail;
+        var img = $(document.createElement('img'));
+          console.log(img);
+          img.attr('src', thumbnail)
```

Comment:
You're pretty close here. Next step will be taking this image and adding it to the DOM, via `setInterval`. You can create another function to do this.